cocos/3d/lod/lodgroup-component.ts

```diff
@@ -628,6 +643,14 @@ export class LODGroup extends Component {
     /**
      * @engineInternal
      */
+    protected _updateLockedLODLevels (): void {
+        if (this._forceUsedLevels.length > 0) {
+            this.lodGroup.lockLODLevels(this._forceUsedLevels);
+        } else {
+            this.lodGroup.clearLockedLODLevels();
+        }
```

Comment:
I think just invoke `this.lodGroup.lockLODLevels(this._forceUsedLevels);` is enough.